autoload/lsp/utils/location.vim

```diff
@@ -1,7 +1,12 @@
 function! s:open_location(path, line, col) abort
     normal! m'
     let l:buffer = bufnr(a:path)
-    if &modified && !&hidden
+    if exists('g:lsp_custom_open_location_fn')
+        call g:lsp_custom_open_location_fn[0]({
+                    \"path":fnameescape(a:path),
```

Comment:
Prefer single quote instead of double quote.

Reply:
Done.